packages/stylelint-config/README.md

```diff
@@ -0,0 +1,35 @@
+# @kununu/stylelint-config
+
+> kununu shareable config for stylelint
+
+For consistent CSS code across kununu's repos.
+
+## Installation
+
+```console
+$ npm install --save-dev @kununu/stylelint-config stylelint
```

Comment:
Can we have here npx install, like eslint-config-kununu/README.md?

Reply:
sure thing